client/src/main/java/io/pravega/client/state/impl/UpdateOrInit.java

```diff
@@ -24,6 +24,7 @@
 import lombok.Data;
 
 @Data
+public
```

Comment:
What are the implications of exposing this class publicly? Also, avoid the line break after `public`.